pyp2rpm/pulpcore.spec

```diff
@@ -0,0 +1,104 @@
+{{ data.credit_line }}
```

Comment:
Wondering if this file should have `.j2` file extension.

Reply:
only if we want `pyp2rpm` not find it ;)

Reply:
(it uses splitext internally, and having two extensions would confuse that)